part1.md

```diff
@@ -417,7 +417,31 @@ software they use must specify the primary SP. Primary SP should be used
 as the only SP to download the data. Users can change the primary SP for
 their objects later if they are not satisfied with their service.
 
-#### 5.2.2 Data Redundancy
+#### 5.2.2 Virtual Group and Virtual Family
+
+A Virtual Group consists of a primary storage provider and several secondary storage providers. 
+Each object is associated with a virtual group, limiting the range of secondary storage providers for storing object 
+replica data. The number of storage providers in each virtual group is determined by the redundancy policy. 
+E.g If our redundancy policy specifies a full replica and a 4+2 erasure coding (EC) replica tree, so every 
```

Comment:
What is `replica tree` or a typo?

Reply:
This is what I learn from Fynn

Reply:
please remove the `tree`. It's a typo. thx.

Reply:
fixed.